DATA_PREPARATION.md

```diff
@@ -200,15 +200,21 @@ For `key_value_pair` dataset, an additional field `schema` is required to define
-| enum        | list[string \| number \| integer] | only work when type is string or number or integer                                                                                                             | no                      |
-| properties  | dict[string, FieldSchema]         | define FieldSchema of each subfield                                                                                                                            | yes when type is object |
-| items       | FieldSchema                       | define the FieldSchema for all items in array                                                                                                                  | yes when type is array  |
-
-More details can be found at [`vision-datasets/vision_datasets/key_value_pair/manifest.py`](vision_datasets/key_value_pair/manifest.py)
+| Property         | Type                      | Details                                                                                                                                 | Required?               |
+| :--------------- | :------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------- | :---------------------- |
+| type             | FieldValueType            | JSON type: string, number, integer, boolean, array, object.                                                                             | yes                     |
+| description      | string                    | describes the field in more detail                                                                                                      | no                      |
+| examples         | list[string]              | examples of field content                                                                                                               | no                      |
+| classes          | dict[str, ClassSchema]    | dictionary that maps each class name to `ClassSchema`.                                                                                  | no                      |
+| properties       | dict[string, FieldSchema] | defines FieldSchema of each subfield                                                                                                    | yes when type is object |
+| items            | FieldSchema               | defines the FieldSchema for all items in array                                                                                          | yes when type is array  |
+| includeGrounding | boolean                   | whether annotation of this field should contain bbox groundings; if true, bboxes are stored in the `groundings` field of the annotation | No, default false       |
```

Comment:
"whether annotation of this field should contain bbox groundings"
=>
"whether this annotation field has bbox groundings associated"